server/routes/api.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express')
+const mongoose = require('mongoose')
+// const data = require('../../data.json')
+const Expense = require('../../model/Expense')
+const moment = require('moment')
+
+const route = express()
```

Comment:
In your api files, you should use the express Router instance, not the express() main instance (usually called app)